traitquestdev/css/style.css

```diff
@@ -15,29 +15,40 @@
 
 /* 2.0 GLOBALS
 ------------------------------------------ */
-html, body{background:#F9F9F9;color:#333333;margin:0;padding:0; height:100%; }
-body{font-family: 'Lato', sans-serif; text-align:justify; }
+html, body{background:#F9F9F9;color:#333333;}
+html{position:relative; min-height:100%;}
+body{font-family: 'Lato', sans-serif; text-align:justify; margin-bottom:40px;}
 a, a:hover, a:active, a:focus {text-decoration:none;}
+a.yellowAnchor{color:#F5D76E;}
+a.yellowAnchor:hover{color:#DCC163;}
 h1,h2,h3,h4,h5,h6 {color:#666666;}
 ul{margin:0; padding: 0; list-style:none;}
 form{margin: 0; padding: 0;}
 input:focus, textarea:focus, select:focus{ outline:none;}
 input[type=number]::-webkit-inner-spin-button,input[type=number]::-webkit-outer-spin-button{-webkit-appearance:none;margin:0;}
-select{}
 textarea{resize:none;}
 #logo-image{max-height:60px;}
+#header-bg{background-image:url('../images/bg-img.jpg');background-size: 100% 100%;}
+#footer-bg{background-color:#666666;margin-bottom:40px;}
+#cyan-bg{background-color:#D5FFFF;}
```

Comment:
the **cyan and dark green** background can be used from color.css

-for some reason, if i change the **footer-bg** ID and add class into the div, the margin and color doesn't appear
-i tried adding grey-bg class, it doesn't apply the margin, but it applies the color
-if it's lightgrey-bg class, both margin and color doesn't apply

for **header-bg** ID, the image is only 1 time use i think, so i used ID for it.